test/cypress/basic_test.lua

```diff
@@ -122,11 +122,20 @@ function g.test_users()
         ',cypress/integration/remove-user.spec.js' ..
         ',cypress/integration/server-details.spec.js' ..
         ',cypress/integration/login-and-logout.spec.js'..
-        ',cypress/integration/auth-switcher-not-moved.spec.js'
+        ',cypress/integration/auth-switcher-not-moved.spec.js'..
+        ',cypress/integration/schema-editor-with-bootstrap.spec.js'
```

Comment:
Please merge it with `schema-editor.spec.js`

Reply:
done